tornjak-frontend/src/index.tsx

```diff
@@ -3,13 +3,21 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import KeycloakService from "./auth/KeycloakAuth";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const renderApp = () =>
+  ReactDOM.render(
+    <React.StrictMode>
+      <App />
+    </React.StrictMode>,
+    document.getElementById('root')
+  );
+
+if(process.env.REACT_APP_AUTH_SERVER_URI) { // with Auth for testing purposes
```

Comment:
is this an environment variable? I see it in a couple other places - is there a way to put them all separately to denote what needs to be set before running? 
Although I suppose it doesn't matter if we are going to be setting these with a call to backend

Reply:
Yes, this is just for Dev to set the Auth Server URI once. But we will be getting this from the backend call.